EXAMPLES/MATRIX_MARKET/Makefile.am

```diff
@@ -1,6 +1,8 @@
 LDADD = $(top_builddir)/SRC/libarpack$(LIBSUFFIX).la
 AM_CPPFLAGS = -I$(top_builddir) -I$(top_srcdir)/ICB $(EIGEN3_CFLAGS)
 
+pkgincludedir = $(includedir)/arpack
```

Comment:
@turboencabulator : this was explicitly "asked for" at some point (#126). Also both autotools/cmake should behave the same way (not sure what cmake does). I have no religion here but I understand the point from #126.

Reply:
This line is for the `pkginclude_HEADERS = arpackSolver.hpp` line below.  Without it, it gets confused on where the file should be installed.
cmake should be equvalent, it has `install(FILES EXAMPLES/MATRIX_MARKET/arpackSolver.hpp DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/arpack)`

Reply:
OK, sure, missed that this was (only) in `Makefile.am` of `EXAMPLES/MATRIX_MARKET`.